### ISSUE TITLE
scylla-master-perf-regression-latency-250gb-with-nemesis: install scylla-manager-agent fails: E: Malformed line 1 in source list /etc/apt/sources.list.d/scylla-manager.list (type)

### ISSUE THREAD

**Issue by yarongilor** [scylladb/scylla-cluster-tests#3410]
build URL": https://jenkins.scylladb.com/job/scylla-master/job/scylla-master-perf-regression-latency-250gb-with-nemesis/46/execution/node/71/log/

error:
```
06:10:02  < t:2021-04-09 03:10:02,278 f:base.py         l:144  c:RemoteCmdRunner      p:ERROR > Error executing command: "sudo bash -cxe "
06:10:02  < t:2021-04-09 03:10:02,278 f:base.py         l:144  c:RemoteCmdRunner      p:ERROR > apt-get install -y scylla-manager-agent
06:10:02  < t:2021-04-09 03:10:02,278 f:base.py         l:144  c:RemoteCmdRunner      p:ERROR > sed -i 's/# auth_token:.*$/auth_token: 53b5a012-367a-4c3e-9056-1ea0516f22a0/' /etc/scylla-manager-agent/scylla-manager-agent.yaml
06:10:02  < t:2021-04-09 03:10:02,278 f:base.py         l:144  c:RemoteCmdRunner      p:ERROR > scyllamgr_ssl_cert_gen
06:10:02  < t:2021-04-09 03:10:02,278 f:base.py         l:144  c:RemoteCmdRunner      p:ERROR > sed -i 's/#tls_cert_file/tls_cert_file/' /etc/scylla-manager-agent/scylla-manager-agent.yaml
06:10:02  < t:2021-04-09 03:10:02,278 f:base.py         l:144  c:RemoteCmdRunner      p:ERROR > sed -i 's/#tls_key_file/tls_key_file/' /etc/scylla-manager-agent/scylla-manager-agent.yaml
06:10:02  < t:2021-04-09 03:10:02,278 f:base.py         l:144  c:RemoteCmdRunner      p:ERROR > sed -i 's/#prometheus: .*/prometheus: :5090/' /etc/scylla-manager-agent/scylla-manager-agent.yaml
06:10:02  < t:2021-04-09 03:10:02,278 f:base.py         l:144  c:RemoteCmdRunner      p:ERROR > systemctl restart scylla-manager-agent
06:10:02  < t:2021-04-09 03:10:02,278 f:base.py         l:144  c:RemoteCmdRunner      p:ERROR > systemctl enable scylla-manager-agent
06:10:02  < t:2021-04-09 03:10:02,278 f:base.py         l:144  c:RemoteCmdRunner      p:ERROR > ""; Exit status: 100
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 2021-04-09 03:10:02.282: (TestFrameworkEvent Severity.ERROR), source=PerformanceRegressionTest.SetUp()
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > exception=[Node perf-latency-nemesis-perf-v10-db-node-53b5a012-1 [18.202.225.167 | 10.0.3.130] (seed: True)] NodeSetupFailed: Encountered a bad command exit code!
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > Command: 'sudo bash -cxe "\napt-get install -y scylla-manager-agent\nsed -i \'s/# auth_token:.*$/auth_token: 53b5a012-367a-4c3e-9056-1ea0516f22a0/\' /etc/scylla-manager-agent/scylla-manager-agent.yaml\nscyllamgr_ssl_cert_gen\nsed -i \'s/#tls_cert_file/tls_cert_file/\' /etc/scylla-manager-agent/scylla-manager-agent.yaml\nsed -i \'s/#tls_key_file/tls_key_file/\' /etc/scylla-manager-agent/scylla-manager-agent.yaml\nsed -i \'s/#prometheus: .*/prometheus: :5090/\' /etc/scylla-manager-agent/scylla-manager-agent.yaml\nsystemctl restart scylla-manager-agent\nsystemctl enable scylla-manager-agent\n"'
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > Exit code: 100
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > Stdout:
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > Stderr:
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > + apt-get install -y scylla-manager-agent
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > E: Malformed line 1 in source list /etc/apt/sources.list.d/scylla-manager.list (type)
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > E: The list of sources could not be read.
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > E: Malformed line 1 in source list /etc/apt/sources.list.d/scylla-manager.list (type)
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > E: The list of sources could not be read.
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > Traceback (most recent call last):
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >   File "/home/jenkins/slave/workspace/scylla-master/scylla-master-perf-regression-latency-250gb-with-nemesis@2/scylla-cluster-tests/sdcm/cluster.py", line 3261, in node_setup
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >     cl_inst.node_setup(_node, **setup_kwargs)
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >   File "/home/jenkins/slave/workspace/scylla-master/scylla-master-perf-regression-latency-250gb-with-nemesis@2/scylla-cluster-tests/sdcm/cluster.py", line 3904, in node_setup
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >     node.install_manager_agent(package_path=pkg_path)
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >   File "/home/jenkins/slave/workspace/scylla-master/scylla-master-perf-regression-latency-250gb-with-nemesis@2/scylla-cluster-tests/sdcm/cluster.py", line 1853, in install_manager_agent
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >     self.remoter.run('sudo bash -cxe "%s"' % install_and_config_agent_command)
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >   File "/home/jenkins/slave/workspace/scylla-master/scylla-master-perf-regression-latency-250gb-with-nemesis@2/scylla-cluster-tests/sdcm/remote/remote_base.py", line 604, in run
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >     result = _run()
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >   File "/home/jenkins/slave/workspace/scylla-master/scylla-master-perf-regression-latency-250gb-with-nemesis@2/scylla-cluster-tests/sdcm/utils/decorators.py", line 61, in inner
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >     return func(*args, **kwargs)
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >   File "/home/jenkins/slave/workspace/scylla-master/scylla-master-perf-regression-latency-250gb-with-nemesis@2/scylla-cluster-tests/sdcm/remote/remote_base.py", line 596, in _run
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >     return self._run_execute(cmd, timeout, ignore_status, verbose, new_session, watchers)
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >   File "/home/jenkins/slave/workspace/scylla-master/scylla-master-perf-regression-latency-250gb-with-nemesis@2/scylla-cluster-tests/sdcm/remote/remote_base.py", line 537, in _run_execute
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >     result = connection.run(**command_kwargs)
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >   File "<decorator-gen-3>", line 2, in run
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >   File "/usr/local/lib/python3.8/site-packages/fabric/connection.py", line 30, in opens
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >     return method(self, *args, **kwargs)
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >   File "/usr/local/lib/python3.8/site-packages/fabric/connection.py", line 721, in run
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >     return self._run(self._remote_runner(), command, **kwargs)
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >   File "/usr/local/lib/python3.8/site-packages/invoke/context.py", line 101, in _run
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >     return runner.run(command, **kwargs)
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >   File "/usr/local/lib/python3.8/site-packages/invoke/runners.py", line 291, in run
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >     return self._run_body(command, **kwargs)
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >   File "/usr/local/lib/python3.8/site-packages/invoke/runners.py", line 442, in _run_body
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  >     raise UnexpectedExit(result)
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > invoke.exceptions.UnexpectedExit: Encountered a bad command exit code!
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > Command: 'sudo bash -cxe "\napt-get install -y scylla-manager-agent\nsed -i \'s/# auth_token:.*$/auth_token: 53b5a012-367a-4c3e-9056-1ea0516f22a0/\' /etc/scylla-manager-agent/scylla-manager-agent.yaml\nscyllamgr_ssl_cert_gen\nsed -i \'s/#tls_cert_file/tls_cert_file/\' /etc/scylla-manager-agent/scylla-manager-agent.yaml\nsed -i \'s/#tls_key_file/tls_key_file/\' /etc/scylla-manager-agent/scylla-manager-agent.yaml\nsed -i \'s/#prometheus: .*/prometheus: :5090/\' /etc/scylla-manager-agent/scylla-manager-agent.yaml\nsystemctl restart scylla-manager-agent\nsystemctl enable scylla-manager-agent\n"'
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > Exit code: 100
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > Stdout:
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > Stderr:
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > 
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > + apt-get install -y scylla-manager-agent
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > E: Malformed line 1 in source list /etc/apt/sources.list.d/scylla-manager.list (type)
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > E: The list of sources could not be read.
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > E: Malformed line 1 in source list /etc/apt/sources.list.d/scylla-manager.list (type)
06:10:02  < t:2021-04-09 03:10:02,292 f:file_logger.py  l:78   c:sdcm.sct_events.file_logger p:INFO  > E: The list of sources could not be read.
```

**Comment by juliayakovlev** [comment]
Still fails https://jenkins.scylladb.com/job/scylla-master/job/scylla-master-perf-regression-latency-250gb-with-nemesis/47/consoleFull

**Comment by roydahan** [comment]
@fgelcer will take care of it.
These jobs are running from branch-perf-v10.
The easy and fast solution will be to add the mgmt_agent_repo paramter to the job and trigger it with the correct agent version.

**Comment by fgelcer** [comment]
yes, i'm on it

**Comment by fgelcer** [comment]
@yarongilor @juliayakovlev @roydahan , here is the PR to fix it #3425

**Comment by fgelcer** [comment]
> @yarongilor @juliayakovlev @roydahan , here is the PR to fix it #3425

now, once the pipeline updates the Jenkins job, we will know if it is fixed (BTW, there is a running job, meaning that the failing poing is behind us)

**Comment by fgelcer** [comment]
#3425 fixed this issue (see official job on master)
